config/environments/development.rb

```diff
@@ -69,6 +69,7 @@
 
   # Allow smee requests
   config.hosts << 'smee.io'
+  config.hosts << 'editor-api.rpfdev.com'
```

Comment:
Not relevant to this PR in particular, just another sneaky addition to facilitate future use of core-dev-services! Hope this is ok :)